src/run_graph.rs

```diff
@@ -3,20 +3,18 @@ use crossbeam;
 use errors::*;
 use ex::{self, ExMode, Experiment};
 use file;
-use petgraph::dot::Dot;
-use petgraph::graph::{Graph, NodeIndex};
-use petgraph::visit::EdgeRef;
-use petgraph::Direction;
-use results::WriteResults;
+use petgraph::{dot::Dot, graph::NodeIndex, stable_graph::StableDiGraph, Direction};
+use results::{TestResult, WriteResults};
 use std::fmt;
 use std::mem;
 use std::path::Path;
-use std::sync::Mutex;
+use std::sync::{Arc, Mutex};
 use tasks::{Task, TaskStep};
+use util;
 
 pub enum Node {
     Task(Task),
-    RunningTask,
+    RunningTask(Option<Arc<Task>>),
```

Comment:
Could we have a comment on this as to the meaning of the Option?

Reply:
Refactored the `Option` away.